windows_containers/windows-containers-release-notes-10-16-x.adoc

```diff
@@ -0,0 +1,101 @@
+
+The WMCO can now use both *ImageDigestMirrorSet* (IDMS) and *ImageTagMirrorSet* (ITMS) objects to pull images from mirrored registries.
+
+// For more information, see xref:	../windows_containers/enabling-windows-container-workloads.adoc#images-configuration-registry-mirror_enabling-windows-container-workloads[Understanding image registry repository mirroring]
+
+[id="wmco-10-16-0-filesystem"]
+==== *Filesystem* metrics now display for Windows nodes
+
+The *Filesystem* metrics are now available for Windows nodes in the *Utilization* tile of the *Node details* page in the {product-title} web console. You can xref:../observability/monitoring/managing-metrics.adoc#about-querying-metrics_managing-metrics[query the metrics] by running Prometheus Query Language (PromQL) queries.  The charts previously reported *No datapoints found*.
+
+[id="wmco-10-16-0-network-charts"]
+==== Pod network metrics now display for the pods on Windows nodes
+
+The *Network in* and *Network out* charts are now available for Windows pods on the *Pod details* page in the {product-title} web console. You can xref:../observability/monitoring/managing-metrics.adoc#about-querying-metrics_managing-metrics[query the metrics] by running PromQL queries. The charts previously reported *No datapoints found*.
+
+[id="wmco-10-16-0-cpu-memory"]
+==== Pod CPU and memory metrics now display for the pods on Windows nodes
+
+The CPU and memory usage metrics are now available for Windows pods on the *Pods* and *Pod details* pages in the {product-title} web console. You can xref:../observability/monitoring/managing-metrics.adoc#about-querying-metrics_managing-metrics[query the metrics] by running PromQL queries. The chart previously reported *No datapoints found*.
+
+[id="wmco-10-16-0-operator-kube"]
+==== Kubernetes upgrade
+
+The WMCO now uses Kubernetes 1.29.
+
+[id="wmco-10-16-0-bug-fixes"]
+=== Bug fixes
+
+Because the WICD service account was missing a required secret, the WMCO was unable to properly configure Windows nodes in a Nutanix cluster. With this fix, the WMCO creates a long-lived token secret for the WICD service account. As a result, the WMCO is able to configure a Windows node on Nutanix. (link:https://issues.redhat.com/browse/OCPBUGS-22680[*OCPBUGS-22680*])
+
+Previously, the WMCO performed a sanitization step that incorrectly replaced commas with semicolons in the user's cluster-wide proxy configuration. This caused Windows to ignore the values set in the `noProxy` environment variable. As a consequence, the WMCO incorrectly sent traffic through the proxy for the endpoints specified in the `no-proxy` parameter. With this fix, the sanitization step that replaced commas with semicolons was removed. As a result, web requests from a Windows node to a cluster-internal endpoint or an endpoint that exists in the `no-proxy` parameter does not go through the proxy. (link:https://issues.redhat.com/browse/OCPBUGS-24264[*OCPBUGS-24264*])
```

Comment:
```suggestion
Previously, the WMCO performed a sanitization step that incorrectly replaced commas with semicolons in a user's cluster-wide proxy configuration. This behavior caused Windows to ignore the values set in the `noProxy` environment variable. As a consequence, the WMCO incorrectly sent traffic through the proxy for the endpoints specified in the `no-proxy` parameter. With this fix, the sanitization step that replaced commas with semicolons was removed. As a result, web requests from a Windows node to a cluster-internal endpoint or an endpoint that exists in the `no-proxy` parameter do not go through the proxy. (link:https://issues.redhat.com/browse/OCPBUGS-24264[*OCPBUGS-24264*])
```
- `a user` because you're referring to a generic user and not a specific user
- [Per ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references), make sure pronoun references are clear by adding a reference for the pronoun "this". "this" what?
- Fix subject verb disagreement `...web requests... do not go`